### PR TITLE
Simplify initdelete classes

### PR DESCRIPTION
**Pull request by leeweiming3**
Command now returns two commands which will be run if the confirm box appears and y/n inputted

### CODE REVIEW

File: src/main/java/seedu/hirehub/logic/LogicManager.java [hunk 1]
```diff
@@ -68,24 +59,16 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
...
+        case CONFIRM:
+            command = confirmationStageParser.parseCommand(commandText);
             break;
         default:
             break;
         }
         assert (command != null);
         commandResult = command.execute(model);
         state = commandResult.getCommandBoxState();
+        confirmationStageParser.setNextCommands(command);
```
Comment (0liverkong) [inline src/main/java/seedu/hirehub/logic/LogicManager.java:71]: its a little weird that the parser has attributes that stores the next commands to be executed right... is there a nicer way to go about this? I cant think of any but if you are able to that'd be great

**Review by 0liverkong — COMMENTED** [review]
lgtm besides the two comments

File: src/main/java/seedu/hirehub/logic/CommandBoxState.java [hunk 2]
```diff
@@ -6,6 +6,7 @@
  */
 public enum CommandBoxState {
     NORMAL,
+    CONFIRM,
     CLEARCONFIRM,
     DELETECONFIRM,
     DELETEAPPLICATIONCONFIRM,
```
Comment (chiralcentre) [inline src/main/java/seedu/hirehub/logic/CommandBoxState.java:12]: what is the purpose of these three enum states? CMIIW, but I don't see them used anywhere after the refactoring

Comment (leeweiming3) [reply src/main/java/seedu/hirehub/logic/CommandBoxState.java:12]: forgot to remove them, fixed

**Comment by 0liverkong** [comment]
lgtm

**Review by 0liverkong — APPROVED** [review]
lgtm

**Pull request merged**